File: array.js
// ARRAY CRUD
const students = ['Azat', 'Yuki', 'Nana'];

// Create
students.push("Devin")
console.log(students);
// Read
console.log(students[1]);

// Update
students[1] = "Super Yuki"; 
console.log(students);

// Delete
students.splice(1, 2) // remove, from index 1, two elements
console.log(students);

// Iterate
// JS arrow function () => {}
students.forEach((student) => {
  console.log(`${student} is amazing!`);
});

File: array.rb
# ARRAY CRUD
students = ['Azat', 'Yuki', 'Nana']

# Create
students << "Kyle"
students.push("Devin")
p students

# Read
p students[1]

# Update
students[1] = "Super Yuki"
p students

# Delete
students.delete_at(1)
students.delete("Kyle")
p students

# Iterate
students.each do |student|
  puts "#{student} is amazing!"
end